src/components/Speciality/BlogPosts.js

```diff
@@ -21,23 +21,26 @@ const BlogPosts = ({ title, posts }) => {
               <ul>
                 {posts
                   .map(({ node }) => node)
-                  .map(({ id, uniqueSlug, title, createdAt }) => (
+                  .map(({ id, uniqueSlug, title, firstPublishedAt }) => (
                     <li key={`${id}`}>
-                      <Subtitle>
+                      <Subtitle noPaddingBottom>
```

Comment:
spacing fixes

Reply:
Designs have a bottom padding :confused: 
![image](https://user-images.githubusercontent.com/15656538/55489497-e9f05380-5629-11e9-8cbb-2ecd6335961f.png)

Reply:
Possibly because it can look a bit weird when there's 2 lines of subtitle:

Reply:
it's strange, it was beatriz that warned me about the extra spacing (which homepage doesn't have and she said that home was flawless..)

Reply:
You're right, homepage was the one that's wrong. Will add the spacing on thome and remove it here 👍